PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -189,7 +189,7 @@ def miniAOD_customizeCommon(process):
 	 valueLabels = cms.vstring('pt','emEnergyFraction'),
 	 lazyParser = cms.bool(True) )
     task.add(process.caloJetMap)
-    process.patJets.userData.userFloats.src += [ cms.InputTag("caloJetMap:pt"), cms.InputTag("caloJetMap:emEnergyFraction") ]
+#    process.patJets.userData.userFloats.src += [ cms.InputTag("caloJetMap:pt"), cms.InputTag("caloJetMap:emEnergyFraction") ]
```

Comment:
Hi@slava77, the pt and emEnergyFraction of caloJetMap are not needed when the calo jet is implemented. Should I just remove this line from the code?